lib/membrane/clock.ex

```diff
@@ -227,10 +222,9 @@ defmodule Membrane.Clock do
   end
 
   defp do_handle_clock_update(till_next, state) do
-    use Ratio
     %{till_next: from_previous, clock_time: clock_time} = state
-    clock_time = clock_time + from_previous
-    ratio = clock_time / (state.time_provider.() - state.init_time)
+    clock_time = Ratio.add(clock_time, from_previous)
+    ratio = Ratio.new(clock_time, state.time_provider.() - state.init_time)
```

Comment:
`div` is less confusing in this case IMO

```suggestion
    ratio = Ratio.div(clock_time, state.time_provider.() - state.init_time)
```

Reply:
done